portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro

```diff
@@ -23,6 +23,15 @@ definition {
 			widgetName = "${widgetName}");
 	}
 
+	@summary = "Asserts a list of items by the name"
+	macro assertItemsName {
```

Comment:
Hi @91nilo , I think this should be change to something like assertAttributeList or assertItemsList. Since the macro is assert a list of items.